modules/price/tests/src/Functional/CurrencyTest.php

```diff
@@ -1,66 +1,79 @@
 <?php
 
-namespace Drupal\commerce_price\Tests;
+namespace Drupal\Tests\commerce_price\Functional;
 
 use Drupal\commerce_price\Entity\Currency;
-use Drupal\commerce\Tests\CommerceTestBase;
+use Drupal\simpletest\BlockCreationTrait;
+use Drupal\Tests\commerce\Functional\CommerceBrowserTestBase;
 
 /**
  * Tests the currency UI.
  *
  * @group commerce
  */
-class CurrencyTest extends CommerceTestBase {
+class CurrencyTest extends CommerceBrowserTestBase {
+
+  use BlockCreationTrait;
 
   /**
    * Modules to enable.
    *
    * @var array
    */
   public static $modules = [
-    'user',
+    'block',
     'commerce_store',
   ];
 
   /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->placeBlock('local_tasks_block');
```

Comment:
Why doesn't CommerceBrowserTestBase place the blocks like CommerceTestBase does? Seems common enough